packages/notifi-graphql/lib/gql/fragments/Web3TargetFragment.gql.ts

```diff
@@ -0,0 +1,34 @@
+import { gql } from 'graphql-request';
+
+export const Web3TargetFragment = gql`
+  fragment Web3TargetFragment on Web3Target {
+    id
+    name
+    accountId
+    walletBlockchain
+    targetProtocol
+    isConfirmed
+  }
+`;
+
+export const GetWeb3TargetResponseFragment = gql`
+  fragment GetWeb3TargetResponseFragment on Web3TargetsConnection {
```

Comment:
Just one question about the naming. @kai-notifi .
As far as know, this is the CB wallet target. Can I understand the intention of  naming it as Web3Target such a general name is that we will finally include all the blockchain related target into this one if there are come coming up??
CC: @ankush-notifi @nimesh-notifi